src/Security/src/Authentication.CloudFoundryCore/CloudFoundryJwtBearerOptions.cs

```diff
@@ -31,5 +31,11 @@
             get { return Validate_Certificates; }
             set { Validate_Certificates = value; }
         }
+
+        public void SetEndpoints(string authDomain)
+        {
+            JwtKeyUrl = (!string.IsNullOrWhiteSpace(authDomain)) ?
+                authDomain + CloudFoundryDefaults.JwtTokenUri : JwtKeyUrl;
```

Comment:
Is Codecov wrong here or is this line somehow missed?

Reply:
I think the Theory -> MemberData with the inserted blanks test may not be getting linked to this for some reason [CloudFoundryJwtBearerOptionsTest.cs - Line 43](https://github.com/SteeltoeOSS/Steeltoe/pull/579/files#diff-e8a0bcf4337802a87e90109b96d839526207cc8b398dea7ccf4a0bcaf91acdc4R43)